scenarioo-client/app/shared/navigation/navigation.controller.js

```diff
@@ -31,6 +31,22 @@ function NavigationController($scope, $location, LocalStorageService, BranchesAn
 
     SelectedBranchAndBuildService.callOnSelectionChange(loadBranchesAndBuildsService);
 
+    $scope.globalSearch = {
```

Comment:
This controller was not migrated to vm yet, this is part of https://github.com/scenarioo/scenarioo/issues/509 and does not need to be changed in this PR.